### ISSUE TITLE
State name is not reported to teamcity

### ISSUE THREAD

**Issue by andrey-skl** [gemini-testing/gemini-teamcity#3]
I have a simple test like:

```
gemini.suite('Alert', function(suite) {
  suite
    .setUrl('/example-alert/index.html')
    .setCaptureElements('body > div')
    .capture('plain')
    .capture('hovered', function(actions, find) {
      actions.mouseMove(find('.alert__close'));
    });
```

Report looks like: 

```
##teamcity[testStarted name='Alert.chrome']
##teamcity[testFinished name='Alert.chrome']
##teamcity[testStarted name='Alert.chrome']
##teamcity[testFinished name='Alert.chrome']
##teamcity[testStarted name='Alert.firefox']
##teamcity[testFinished name='Alert.firefox']
##teamcity[testStarted name='Alert.firefox']
##teamcity[testFinished name='Alert.firefox']
```

As you can see, suite name only is represented in test names, but state name (like "plain", "hovered") is missed.

Thats why teamcity thinks that I had ran all tests twice.

**Comment by Saulis** [comment]
This was a total rookie mistake on my behalf. Somehow I had in my mind thought suite.name == state.name but obviously that isn't the case. 

I'll fix this in a sec.

**Comment by Saulis** [comment]
Fixed in 1.0.1

**Comment by andrey-skl** [comment]
@Saulis thanks, works like a charm)
